### ISSUE TITLE
MiniCart: title inconsistency

### ISSUE THREAD

**Issue by kqmpetenz** [Xceptance/posters-demo-store#209]
Issue 1: Wrong MiniCart title after adding item to empty cart.

Reproduce:
1. make sure cart is empty
2. add exactly 1 product to cart (ProductDetailPage)
3. open MiniCart

When adding a new item to an empty shopping cart, the title of the MiniCart will be displayed wrongly by saying "1 ITEMS IN CART":
![1](https://github.com/Xceptance/posters-demo-store/assets/65414932/91038c0e-a67d-42f0-95c6-be5b2769d86e)
Only after updating the page by for example going to the CartPage, the title of the MiniCart will be displayed correctly:
![2](https://github.com/Xceptance/posters-demo-store/assets/65414932/1a67fc62-a5bd-4d07-8211-be31492448d2)

Issue 2: Wrong MiniCart title after changing the quantity of a specific item on CartPage.

Reproduce:
1. make sure cart is empty
2. add exactly 1 product to cart (ProductDetailPage)
4. change quantity of item in cart to more than 1
5. open MiniCart

When changing the quantity of an item on the CartPage to something bigger than 1, the title of the MiniCart will be displayed wrongly by saying "5 ITEM IN CART":
![3](https://github.com/Xceptance/posters-demo-store/assets/65414932/3243bc52-7435-4444-817e-e0049cb6d653)
Only updating the page fixes that issue as well. After going to any CategoryPage for example, the title of the MiniCart will then be displayed correctly:
![4](https://github.com/Xceptance/posters-demo-store/assets/65414932/4752b365-edd8-4943-9024-85b8f5ab8e54)

**Comment by freddyka** [comment]
Issue not fixed for poster v2.1.

Note:
1.add 2 or more items to cart
2.remove them
3.add 1 item to cart --> display is now "1 items" not "item" issue is fixed after refreshing the page

![error adding one item after 2 where inside minicart](https://github.com/Xceptance/posters-demo-store/assets/149577992/abdaa325-088b-41ac-b3b6-01234630fde1)

**Comment by EliasTatX** [comment]
As of v2.2 this is still the case. Additional notes: the minicart is created based on a freemarker template in the html and that is only loaded on page load. The template sets the title to "item" if there is exactly 1 item in the cart if there are more or less items in there it uses the default line with "items" since that ("0 items in cart") would be the correct way to say it. The dynamic add to cart that shows the minicart pop up and makes it update without a reload is based on JavaScript and AJAX. We currently do not test in this script how many products in the cart the performed operation is going to result in and accordingly can't adjust the text. If requested this should not be complicated to implement but comes at a slight cost for performance.

**Comment by EliasTatX** [comment]
This ticket is a duplicate of already existing ticket #129
